.github/workflows/ci.yml

```diff
@@ -24,19 +28,16 @@ jobs:
           python-version: ${{ matrix.python-version }}
 
       - name: Install dependencies
-        shell: bash -l {0}
```

Comment:
Why are these not needed

Reply:
Those aren't needed for each action now because I added the following as a default:
```yaml
defaults:
  run:
    shell: bash -l -eo pipefail {0}
```

Reply:
good to know!

Reply:
good to know